bulbs/api/views.py

```diff
@@ -450,8 +450,8 @@ def list(self, request):
                 results.append(dict(
                     name=name,
                     doctype=doctype
-                ))
-        results.sort(key=lambda x: x["name"])
+                    ))
+                results.sort(key=lambda x: x["name"])
```

Comment:
This is a significant whitespace change that should also be reverted, as it will cause sort to be called multiple times instead of just once after for-loop completes (unless this is intended?)